cmd/podman/client.go

```diff
@@ -3,9 +3,10 @@ package main
 import "github.com/containers/podman/v5/libpod/define"
 
 type clientInfo struct {
-	OSArch   string `json:"OS"`
-	Provider string `json:"provider"`
-	Version  string `json:"version"`
+	OSArch      string `json:"OS"`
+	Provider    string `json:"provider"`
+	Version     string `json:"version"`
+	BuildOrigin string `json:"buildOrigin,omitempty" yaml:",omitempty"`
```

Comment:
Why has this a yaml tag as only field?

Reply:
We yaml marshall these on output, and buildOrigin is the only omitempty field we need. For consistency's sake, I can add it to all the fields.